hw/ip/clkmgr/dv/cov/clkmgr_manual_cov_unr_excl.el

```diff
@@ -0,0 +1,49 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This excludes toggle coverage for output ports driven by constant and inputs
+// tied to constants.
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Aug 11 12:43:08 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2617278896 3012009448"
+INSTANCE: tb.dut
```

Comment:
perhaps @tjaychen can take a look at these exclusions

Reply:
They are trivial: the idle_cnt instances are all like this:
  prim_count #(
    .Width(IdleCntWidth)
  ) u_idle_cnt (
    .clk_i(clk_i),
    .rst_ni(rst_ni),
    // the default condition is to keep the clock enabled
    .clr_i(mubi4_test_false_loose(idle_i)),
    .set_i('0),
    .set_cnt_i('0),
    .incr_en_i(mubi4_test_true_strict(idle_i) & ~idle_valid),
    .decr_en_i(1'b0),
    .step_i(IdleCntWidth'(1'b1)),
    .cnt_o(idle_cnt),
    .cnt_next_o(),
    .err_o(cnt_err)
  );
So step_i is tied to IdleCntWidth'(1'b1) and will never toggle.

The various cg_en_o exclusions are driven like this:
  assign cg_en_o.main_powerup = MuBi4False;